aws-rds-dbclusterparametergroup/src/main/java/software/amazon/rds/dbclusterparametergroup/BaseHandlerStd.java

```diff
@@ -359,12 +494,29 @@ private ProgressEvent<ResourceModel, CallbackContext> modifyParameters(
     }
 
     protected boolean isDBClustersAvailable(final ProxyClient<RdsClient> proxyClient, final ResourceModel model) {
-        return proxyClient
-                .injectCredentialsAndInvokeIterableV2(Translator.describeDbClustersRequest(), proxyClient.client()::describeDBClustersPaginator)
-                .stream()
-                .flatMap(describeDbClustersResponse -> describeDbClustersResponse.dbClusters().stream())
-                .filter(dbCluster -> dbCluster.dbClusterParameterGroup().equalsIgnoreCase(model.getDBClusterParameterGroupName()))
-                .allMatch(dbCluster -> dbCluster.status().equals(AVAILABLE));
+        String marker = null;
+        int page = 1;
+        final DescribeDbClustersRequest request = Translator.describeDbClustersRequestForDBClusterParameterGroup(model);
+
+        do {
```

Comment:
Code similar to  this one is repeated a number of times between this resource and DBParameterGroup. Does it makes sense to move this part to common?

Reply:
My attempt to implement a generic method for this operation failed due to a limitation of `AwsResponse`. In particular, `marker` is defined as a member of `SDK_FIELDS` in the corresponding response classes and won't be visible at the level of an abstract class like `AwsResponse`. Given we can not enforce an ad-hoc contract, this turns such a method in a set of functional interface object invocations. Given the long-term approach is to get back to the paginators once we have a reasonable solution, I'd say a little copy-paste is better than a non-sense abstraction that is there to hide the existence of a while-loop.